pkgs/servers/zoneminder/default.nix

```diff
@@ -89,6 +89,8 @@ in stdenv.mkDerivation rec {
 
   patches = [
     ./default-to-http-1dot1.patch
+    # Explicitly link with dynamic linking library to fix build
+    ./link-with-libdl.patch
```

Comment:
Instead of adding the .patch to the tree, please just fetchpatch it from ```https://github.com/ZoneMinder/zoneminder/commit/cda9070389a58450da1616ac865a43d68af3c609.patch```.

Reply:
I wanted to but it doesn't work, their master has advanced from the release and the patch doesn't apply anymore ;)

Reply:
Is that alright?

Reply:
Sure